lib/index.js

```diff
@@ -74,6 +74,14 @@ HubSpot.prototype.identify = function(identify) {
 
 HubSpot.prototype.track = function(track) {
   var props = track.properties();
```

Comment:
You _should_ be able to do `track.properties({ id: '_id' })` here to do the remapping, then just assign the event name on the next line. Slightly cleaner